packages/node_modules/@webex/react-redux-spark/src/sdk.test.js

```diff
@@ -13,7 +13,7 @@ describe('SDK', () => {
       expect(instance.config.appName).toStrictEqual('webex-widgets-test-app');
     });
   });
-  describe('createSDKInstanceProxy', () => {
+  describe.skip('createSDKInstanceProxy', () => {
```

Comment:
If you're removing the code for this function, you should remove the tests as well, not just skip them